### ISSUE TITLE
oro_reminder table does not exist in migrations (fresh installation OroCommerce 3.1.15)

### ISSUE THREAD

**Issue by sundbry** [oroinc/crm#347]
While installing a fresh OroCommerce-CE 3.1.15:
`php ./bin/console oro:install --env=dev --drop-database --timeout=900`

It looks like the script is trying to run a migration on a table that doesn't exist. 

```
  > Oro\Bundle\CRMBundle\Migrations\Schema\OroCRMBundleInstaller
    ERROR: An exception occurred while executing 'UPDATE oro_reminder SET related_entity_classname = REPLACE(related_entity_classname, 'OroCRM', 'Oro')
                    WHERE related_entity_classname LIKE '%OroCRM%'':

SQLSTATE[42S02]: Base table or view not found: 1146 Table 'mydb.oro_reminder' doesn't exist
```

[oro-install-error.txt](https://github.com/oroinc/crm/files/3834539/oro-install-error.txt)

I don't find any other references to the `oro_reminder` table, perhaps this migration is obsolete?

**Comment by sundbry** [comment]
It looks like this migration more properly belongs in https://github.com/oroinc/platform/tree/master/src/Oro/Bundle/ReminderBundle/Migrations/Schema with the rest of the reminder schema evolution

**Comment by anyt** [comment]
This is not a regular schema migration, but it updates data records that are related to OroCRM, so we can't place the migration to the platform package, as th platform should not know about OroCRM.

**Comment by Baachi** [comment]
If someone also entered this issue, this might help. In my case, the `installed` parameter in the `paramters.yml` wasn't set to `null` so Oro thinks it was an upgrade, not a fresh install.

**Comment by mbessolov** [comment]
@Baachi yes, correct. The installer oro:install after detecting that the application is already installed, shows a warning but does not stop. This will be fixed (not stopping after the warning).

In either case you need to follow the instructions contained in the warning before trying to reinstall an already installed application:

> ATTENTION: Oro Application already installed.
> To proceed with install:
>  - set parameter installed: false in config/parameters.yml
>  - remove caches in var/cache folder manually
>  - drop database manually or reinstall over existing database
> ...
> 

_internal ticket ID: BAP-19576_
